images/extras/esxi-6.5

```diff
@@ -55,8 +56,9 @@ esxcli network ip interface ipv4 set -i vmk0 \
 esxcli network ip dns server add --server=\$(cat /vmfs/volumes/cidata/META_DAT|awk '/gateway/ {print \$2}')
 
 # Hackish
-cat /vmfs/volumes/cidata/USER_DAT | awk '/ssh-rsa / {print "ssh-rsa " \$3}' > /etc/ssh/keys-root/authorized_keys
+cat /vmfs/volumes/cidata/USER_DAT | awk '/ssh-rsa / {print \"ssh-rsa \" \$3}' > /etc/ssh/keys-root/authorized_keys
```

Comment:
I tried run awk command:
```
awk: cmd. line:1: /ssh-rsa / {print \"ssh-rsa \" \$3}
awk: cmd. line:1:                   ^ backslash not last character on line
awk: cmd. line:1: /ssh-rsa / {print \"ssh-rsa \" \$3}
awk: cmd. line:1:                   ^ syntax error
```

Reply:
yes because this block is in an echo. i know, it's a bit strange.